target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -587,6 +587,17 @@ define Device/tplink_tl-wpa8630-v1
 endef
 TARGET_DEVICES += tplink_tl-wpa8630-v1
 
+define Device/tplink_tl-wpa8630p-v1
+  $(Device/tplink-8mlzma)
+  SOC := qca9563
+  DEVICE_MODEL := TL-WPA8630P
+  DEVICE_VARIANT := v1
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca988x-ct
+  TPLINK_HWID := 0x86310001
+  SUPPORTED_DEVICES += tl-wpa8630
```

Comment:
This can be dropped, this device was not supported in ar71xx.

Reply:
@eliaspoliceno
> How can i generate an image for tplink_tl-wpa8630p-v1 device without the region you said to drop under generic-tp-link.mk file?

This comment only applies to the SUPPORTED_DEVICES line, the rest of the section would remain.

Reply:
Ok thanks!